src/app_service/service/language/service.nim

```diff
@@ -43,3 +43,11 @@ proc setLanguage*(self: Service, locale: string) =
 
   singletonInstance.localAppSettings.setLocale(locale)
   singletonInstance.engine.setTranslationPackage(joinPath(self.i18nPath, fmt"qml_{locale}.qm"), self.shouldRetranslate)
+
+proc setIsDDMMYYDateFormat*(self: Service, isDDMMYYDateFormat: bool) =
+  if(isDDMMYYDateFormat != singletonInstance.localAppSettings.getIsDDMMYYDateFormat()):
+    singletonInstance.localAppSettings.setIsDDMMYYDateFormat(isDDMMYYDateFormat)
```

Comment:
@noeliaSD no need to put this in service, since the whole logic of `setIsDDMMYYDateFormat` and `setIs24hTimeFormat` is to update the entry in local .ini file, nothing else. Such changes is completely fine to do from the `module.nim` of your module (in this case `src/app/modules/main/profile_section/language/module.nim`) cause the logic is kept there. Feel free to remove changes done in service and controller.